Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
-name = "transform-output"
+name = "rust_test_runner"
```

Comment:
The rename reflects that all test running logic is now in rust, not just the transformation of cargo's output.